Bottom: 46d2c09b14ffdd019ea7f40457e85e20b4dc1fa2
Top:    251ce87c0cff93e0365c1bd08504d77240b0f0c8
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-01-21 15:47:23 -0800

Refresh of doc-poll-time

---

diff --git a/man/rsocket.7 b/man/rsocket.7
index 7b49e46..9cc33f5 100644
--- a/man/rsocket.7
+++ b/man/rsocket.7
@@ -1,4 +1,4 @@
-.TH "RSOCKET" 7 "2010-07-19" "librdmacm" "Librdmacm Programmer's Manual" librdmacm
+.TH "RSOCKET" 7 "2013-01-21" "librdmacm" "Librdmacm Programmer's Manual" librdmacm
 .SH NAME
 rsocket \- RDMA socket API
 .SH SYNOPSIS
@@ -33,13 +33,16 @@ rsetsockopt, rgetsockopt, rfcntl
 Functions take the same parameters as that used for sockets.  The
 follow capabilities and flags are supported at this time:
 .P
-PF_INET, PF_INET6, SOCK_STREAM, IPPROTO_TCP, TCP_MAXSEG
+PF_INET, PF_INET6, SOCK_STREAM, SOCK_DGRAM
 .P
-MSG_DONTWAIT, MSG_PEEK
+SOL_SOCKET - SO_ERROR, SO_KEEPALIVE (flag supported, but ignored),
+SO_LINGER, SO_OOBINLINE, SO_RCVBUF, SO_REUSEADDR, SO_SNDBUF
+.P 
+IPPROTO_TCP - TCP_NODELAY, TCP_MAXSEG
 .P
-SO_REUSEADDR, TCP_NODELAY, SO_ERROR, SO_SNDBUF, SO_RCVBUF
+IPPROTO_IPV6 - IPV6_V6ONLY
 .P
-O_NONBLOCK
+MSG_DONTWAIT, MSG_PEEK, O_NONBLOCK
 .P
 Rsockets provides extensions beyond normal socket routines that
 allow for direct placement of data into an application's buffer.
